src/Microsoft.Extensions.Configuration/DefaultConfigurationBuilderExtensions.cs

```diff
@@ -21,7 +21,7 @@ public static IConfigurationBuilder Include(this IConfigurationBuilder configura
             {
                 throw new ArgumentNullException(nameof(configurationBuilder));
             }
-            configurationBuilder.Add(new IncludedConfigurationProvider(includedConfiguration));
+            configurationBuilder.Add(new IncludedConfigurationSource { Source = includedConfiguration });
```

Comment:
We should cut chaining configuration altogether. It was only needed because of Hosting.dll, which no longer requires it.